src/PathMaster.ts

```diff
@@ -9,7 +9,7 @@ import { Config } from "./Config";
  * If a import url isn't relative
  * and only has ascii + @ in the name it is considered a node module
  */
-const NODE_MODULE = /^([a-z@].*)$/;
+const NODE_MODULE = /^[^:]([a-z@].*)$/;
```

Comment:
And I think that even if that is fixed, this file is going to go further, breaking other things ...